smart_fetch/resources.py

```diff
@@ -37,3 +37,75 @@
     BINARY,
     MEDICATION,
 }
+
+# These are the fields to search for to ask "when was this record created?"
+# (i.e. the administrative date, not the clinical date for "when did this described event happen?")
+# Would be nice if there were a `meta.created` field.
+# If you update this, update the get_created_date call below too.
```

Comment:
not saying do this, necesarily, but this is starting to feel if you combined these two together you'd have a nice lil utility class.

Reply:
The `resources.py` file in general feels like a nice lil utility class :smile: but maybe it could be chopped up and classified further. I'll put that in the back of my mind as I make further edits there.